equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/service/CrmFileService.java

```diff
@@ -34,6 +34,8 @@ public class CrmFileService {
     public static final int CRM_TYPE_7 = 5;
     public static final int CRM_TYPE_14 = 6;
     private static final String CHAR_NULL = "\u0000";
+    private static final String CHAR_QUOTE = " \" ";
```

Comment:
I think we need to replace it as `"\""` otherwise they will only replace cases like " "this" " -> " \"this\" " but not cases like ""that"" -> ""that""; and we want ""that"" to be "\"that\""

Reply:
xml should still have quotes e.g in the below line

<linkedAttachment name="0156 PNC.GORDON
2021-10-29.rtf" fileName="2958322.att"
savetype="OFDAttachRef" />

if we replace quotes then xml will not be wellformed resulting in validation error.

This is a specific case where quote is prefixed and suffixed with space.

Reply:
I see... because if not it will replace the tags. Makes sense then